alacritty/src/display.rs

```diff
@@ -386,6 +393,13 @@ impl Display {
         let physical = PhysicalSize::new(self.size_info.width as u32, self.size_info.height as u32);
         self.window.resize(physical);
         self.renderer.resize(&self.size_info);
+
+        // Set subpixel anti-aliasing.
+        #[cfg(target_os = "macos")]
+        {
+            let use_thin_strokes = config.font.use_thin_strokes();
+            set_font_smoothing(use_thin_strokes);
+        }
```

Comment:
As @kchibisov has stated, this setting requires a restart, so there's no point in trying to 'live reload' it.

We should probably still set it so that the user doesn't have to restart twice, but no need to try and refresh the font.

Reply:
We still should handle reload of this setting, since we should prevent double restart to apply, so putting it in `reload_config` is a way to go anyway. However we should toggle it on start as well somewhere.

Reply:
Yeah exactly, just without reloading the font itself.

Reply:
Just to sum up so we can agree (or disagree):
I'll call it on `reload_config` to ensure we don't get into a double-restart in order to enable it. I'll also call it initially to check if it was changed while Alacritty is closed.

Is the two places im calling it right now ok, or how do I move from here? I'll update the `alacritty.yml` file meanwhile as well.

Reply:
They're not ok. As previously mentioned, `handle_update` is not always called when the config is changed. It should be called from `reload_config`.